Resources/Scripts/Gui/Preferences.as

```diff
@@ -645,7 +645,8 @@ namespace spades {
                                     array<int> = {2, 1, 0});
 
             layouter.AddHeading(_Tr("Preferences", "Feedbacks"));
-            layouter.AddToggleField(_Tr("Preferences", "Chat Notify Sounds"), "cg_chatBeep");
+            layouter.AddSliderField(_Tr("Preferences", "Chat Notify Sounds"), "cg_chatBeep",
+                                    0, 100, 1, ConfigNumberFormatter(0, "%"));
```

Comment:
Add a new constructor overload to `ConfigNumberFormatter` to take a new parameter `prescale`. Use it to rescale the displayed value to a 0–100% range.